server/.golangci.yml

```diff
@@ -129,8 +129,6 @@ issues:
         channels/app/team.go|\
         channels/app/team_test.go|\
         channels/app/upload.go|\
-        channels/app/upload_test.go|\
-        channels/app/user_test.go|\
```

Comment:
@hanzei learning question: would this cause the errcheck linter not to run on these 2 files?

Reply:
The opposite is true. These changes remove the 2 files from the ignore list, allowing the linter to check them.

Reply:
Ah, would not have known that!  Thanks for the answer.